posts/api/serializers.py

```diff
@@ -0,0 +1,73 @@
+from rest_framework.serializers import ModelSerializer
+
+from posts.models import Post
+
+
+class PostCreateSerializer(ModelSerializer):
+    class Meta:
+        model = Post
+        fields = [
+            'title',
+            'description',
+            'category',
+            'author',
+            'price',
+            'isSold',
+            'onDiscount',
+            'discountPercent',
+            'age',
+            'brand',
+            'condition',
+        ]
+
+
+class PostDetailSerializer(ModelSerializer):
+    class Meta:
+        model = Post
+        fields = [
+            'id',
```

Comment:
Would this `id` field be the _UUID_ field?

Reply:
Yes exactly